homeassistant/components/rachio/switch.py

```diff
@@ -509,3 +544,70 @@ async def async_added_to_hass(self) -> None:
                 self.hass, SIGNAL_RACHIO_SCHEDULE_UPDATE, self._async_handle_update
             )
         )
+
+
+class RachioValve(CoordinatorEntity[RachioUpdateCoordinator], SwitchEntity):
+    """Representation of one smart hose timer valve."""
+
+    def __init__(
+        self, person, base, data, coordinator: RachioUpdateCoordinator
+    ) -> None:
+        """Initialize a new smart hose valve."""
+        super().__init__(coordinator)
+        self._person = person
+        self._base = base
+        self.id = data[KEY_ID]
+        self._attr_name = data[KEY_NAME]
+        self._attr_unique_id = f"{self.id}-valve"
+        self._static_attrs = data[KEY_STATE][KEY_REPORTED_STATE]
+        self._attr_is_on = KEY_CURRENT_STATUS in self._static_attrs
+        self._attr_device_info = DeviceInfo(
+            identifiers={
+                (
+                    DOMAIN_RACHIO,
+                    self.id,
+                )
+            },
+            connections={(dr.CONNECTION_NETWORK_MAC, self._base.mac_address)},
```

Comment:
Are these different than the one that is in `entity.py`? If its the same device we are going to have a conflict

Reply:
The connections and ID are different, only similarities would be domain, manufacturer and url.